analyzer/javacg-opal/src/main/java/eu/fasten/analyzer/javacgopal/OPALPlugin.java

```diff
@@ -76,9 +79,18 @@ public void consume(String kafkaRecord, ProcessingLane l) {
                 // Generate CG and measure construction duration.
                 logger.info("[CG-GENERATION] [UNPROCESSED] [-1] [" + mavenCoordinate.getCoordinate() + "] [NONE] ");
                 long date = json.optLong("releaseDate", -1);
-				this.graph = OPALPartialCallGraphConstructor.createPartialJavaCG(mavenCoordinate,
-                        CHA, date, artifactRepository, ONLY_STATIC_CALLSITES);
-                long duration = currentTimeMillis() - startTime; 
+                try {
+                    this.graph = OPALPartialCallGraphConstructor.createPCGFromLocalJar(mavenCoordinate, date,
+                            Paths.get(baseDir, mavenCoordinate.toPath()).toString(),
+                            CHA, ONLY_STATIC_CALLSITES);
+                } catch (FileNotFoundException e) {
+                    logger.warn(e.getMessage());
```

Comment:
When logging, make sure that you preserve the stack trace.

As elaborated before, we can just remove this catch handling for now...